labelImg.py

```diff
@@ -83,9 +82,14 @@ def __init__(self, default_filename=None, default_prefdef_class_file=None, defau
         self.settings.load()
         settings = self.settings
 
+        self.image_data = None
+        self.label_file = None
+
         # Load string bundle for i18n
         self.string_bundle = StringBundle.get_bundle()
-        get_str = lambda str_id: self.string_bundle.get_string(str_id)
```

Comment:
Rationale: Fix warning that lambas should not be assigned, instead define a function